Likely bug in ReadStreamSQLite.h

Compiling with clang, the following warning popped up. Could be a false positive, but it does look
like you are missing some parens:
```
./ReadStreamSQLite.h:60:97: warning: operator '?:' has lower precedence than '+'; '+' will be evaluated first [-Wparentheses]
        std::unique_ptr<RamDomain[]> tuple = std::make_unique<RamDomain[]>(arity + isProvenance ? 2 : 0);
```

--- a/src/ReadStreamSQLite.h
+++ b/src/ReadStreamSQLite.h
@@ -57,7 +57,7 @@ protected:
             return nullptr;
         }
 
-        std::unique_ptr<RamDomain[]> tuple = std::make_unique<RamDomain[]>(arity + isProvenance ? 2 : 0);
+        std::unique_ptr<RamDomain[]> tuple = std::make_unique<RamDomain[]>(arity + (isProvenance ? 2 : 0));
 
         uint32_t column;
         for (column = 0; column < arity; column++) {

